payjoin-cli/src/app/v2.rs

```diff
@@ -128,10 +129,14 @@ impl AppTrait for App {
             .send()
             .await
             .map_err(map_reqwest_err)?;
-        let res = payjoin_proposal
+        let _res = payjoin_proposal
```

Comment:
Is this rename just to indicate that the parsed result isn't used? Just trying to learn idiomatic Rust

Reply:
yes